package.json

```diff
@@ -24,7 +25,7 @@
     "rollup-plugin-node-resolve": "3.3.0",
     "rollup-plugin-typescript2": "0.13.0",
     "rollup-plugin-uglify": "~3.0.0",
-    "tslint": "~5.11.0",
+    "tslint": "^5.17.0",
```

Comment:
You changed the version. So `yarn.lock` will be updated automatically. You need to add that change too.